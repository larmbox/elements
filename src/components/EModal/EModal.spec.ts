import { mount } from '@vue/test-utils';
import { beforeAll, expect, test } from 'vitest';
import { createElements, EModal } from '../..';

beforeAll(() => {
  createElements();
});

test('mount component', async () => {
  expect(EModal).toBeTruthy();

  const wrapper = mount(EModal, {
    props: {
      id: 'modal',
    },
  });

  expect(wrapper.html()).toMatchSnapshot();
  expect(wrapper.html()).toContain('modal');
});
