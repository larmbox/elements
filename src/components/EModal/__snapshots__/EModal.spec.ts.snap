// Vitest Snapshot v1

exports[`mount component 1`] = `
"<div id=\\"e-modal-ref\\" style=\\"display: inline;\\">
  <!--v-if-->
</div>"
`;
